four-node-cluster/setup/resources/is.conf

```diff
@@ -46,5 +46,5 @@ server {
         proxy_set_header Upgrade $http_upgrade;
         proxy_set_header Connection "upgrade";
     }
-    keepalive_requests 4;
+    keepalive_requests 100;
```

Comment:
Modify single node value as well

Reply:
https://github.com/wso2/performance-is/pull/211